### DISCUSSION TITLE
Error receiving too much data

### DISCUSSION THREAD

**Discussion by hallanko** [NightscoutFoundation/xDrip#2077] [Q&A]
Libre2 + Miaomiao3 + Samsung A51 phone (and finally Galaxy watch4 once I get the readings show up on the phone). The trouble is as stated above, "receiving too much data" and no readings on the app. Miaomiao connects and everything else seems to work. What do I do? 

![Screenshot_20220413-122357_xDrip+](https://user-images.githubusercontent.com/103568684/163145442-3bf8286b-c565-4cfb-9016-52b493cdff9c.jpg)
![Screenshot_20220413-122409_xDrip+](https://user-images.githubusercontent.com/103568684/163145471-05db3efe-737d-41f7-be5e-29cdd11b0e68.jpg)

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
@hallanko which L2 sensor is that? Europe?

**Comment by PhilTU** [comment]
Error at 12:16 and 12:21 say OOP2 app not found. OOP2 required to decode a Libre 2 sensor data. Install OOP2.apk open it and exclude it from power saving.
